modules/event_shovel_ecs_service/iam.tf

```diff
@@ -22,6 +22,10 @@ EOF
 }
 
 resource "aws_iam_role" "ecs_executionrole" {
+  # Warning: load bearing string!
+  # Role names matching this pattern are permitted to download the image from the shared event shovel docker repo
```

Comment:
What is the rule we are matching on? Is it a whitelisted suffix?

Reply:
That's why I linked the CPI PR lol.

Anyway, here's the direct code reference: https://github.com/nulogy/Common-Platform-Interface/blob/master/infrastructure/terraform/modules/event_shovel_shared_ecr/policies.tf#L7